doc/rst/language/spec/procedures.rst

```diff
@@ -681,14 +681,14 @@ overloads (see :ref:`Return_Intent_Overloads`).
 Default Intent for ’owned’ and ’shared’
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The default intent for ``owned`` and ``shared`` arguments is
-``const ref``. Arguments can use the ``in`` or ``const in`` intents to
-transfer or share ownership if those arguments apply to ``owned`` or
-``shared`` types.
+The default intent for :type:`~OwnedObject.owned` and
+:type:`~SharedObject.shared` arguments is ``const``. Arguments can use the
+``in`` or ``const in`` intents to transfer or share ownership if those arguments
+apply to :type:`~OwnedObject.owned` or :type:`~SharedObject.shared` types.
```

Comment:
Suggestion: "Arguments can use ..." --> 

To transfer the ownership from an `owned` actual argument or to share the ownership with a `shared` actual argument, the formal argument can use the `in`` or ``const in`` intent.